src/ui/components/shared/UserSettingsModal/UserSettingsModal.tsx

```diff
@@ -28,9 +29,9 @@ function Support() {
           <SettingsBodyHeader>Join us on Discord</SettingsBodyHeader>
           <div className="description">
             Come chat with us on our{" "}
-            <a href="https://discord.gg/n2dTK6kcRX" target="_blank" rel="noopener noreferrer">
+            <Anchor href="https://discord.gg/n2dTK6kcRX" target="_blank" rel="noopener noreferrer">
```

Comment:
Maybe you forgot to remove the `target` and `rel` here?

Reply:
Fixed the type to catch this!